### DISCUSSION TITLE
Export/migrate content from wiki.js to another wiki platform

### DISCUSSION THREAD

**Discussion by mittkkoo** [requarks/wiki#6231] [Help / Questions] [answered]
Is there a nice way to export the entire wiki in a way that can be easily imported to another platform? I want to have a fallback solution for the case, that at some point I have to migrate from wiki.js for whatever reason.

Let's take the scenario migration to Mediawiki, just as a reference.
I tested the "Export to tarball" tool. It exports to JSON and splits the pages, the history, the comments and the assets to different places. The rendered pages in HTML are also in the JSON. Maybe somewhere else I will find the CSS. Now, the next step would be to convert this to XML, readable by Mediawiki. Is there a workflow for this, maybe some converter? Or is there a better way to export to standard format?

**Comment by NGPixel** [comment, marked as answer]
Enable a storage module (e.g. Git or Local Disk) to automatically export content to an external location. It will be exported in markdown or HTML depending on the editor used, which can be imported by pretty much any other software.

**Reply by fguisso** [reply]
awesome, very easy to export from wiki.js to Hugo, thanks
